Makefile

```diff
@@ -13,26 +14,35 @@ define compile_service
 endef
 
 define make_docker
+	$(eval svc=$(subst docker_,,$(1)))
+
 	docker build \
 		--no-cache \
-		--build-arg SVC=$(subst docker_,,$(1)) \
+		--build-arg SVC=$(svc) \
 		--build-arg GOARCH=$(GOARCH) \
 		--build-arg GOARM=$(GOARM) \
-		--tag=mainflux/$(subst docker_,,$(1))-$(2) \
+		--tag=mainflux/$(svc)-$(2) \
```

Comment:
I would remove this arch suffix and leave just `--tag=mainflux/$(svc) \`

Reply:
We can do this, but then if you start generation of ARM images on your PC to copy them on your gateway, your current x86 images will be overwritten.

MAybe this can be useful, as you do not have to retag images once you copy them to the gateway, but them compiling for different architectures can be somehow more difficult...

Reply:
After `make dockers`  you will have double all images with this approach (one with and one without arch suffix). This could be confusing for most users.

Reply:
Size is not doubled, just the tags. Same docker layers are used.

Let me think if we should remove this, but imagine the situation in which we have to build for several architectures (ARM. MIPS, ...). We would have to backup all images prior to build, as they would be overwritten...